_docker/awestruct/Dockerfile

```diff
@@ -16,12 +16,16 @@ RUN wget http://prdownloads.sourceforge.net/optipng/optipng-0.7.5.tar.gz?downloa
 WORKDIR /tmp/optipng-0.7.5
 RUN ./configure && make && make install
 
+RUN yum -y install gcc gcc-c++ make flex bison gperf \
+  openssl-devel freetype-devel fontconfig-devel libicu-devel sqlite-devel \
+  libpng-devel libjpeg-devel
+
 WORKDIR /tmp
 RUN yum install -y fontconfig freetype && \
-    wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-x86_64.tar.bz2 && \
-    mkdir -p /opt/phantomjs && \
-    tar -xjvf /tmp/phantomjs-1.9.8-linux-x86_64.tar.bz2 -C /opt/phantomjs/ && \
-    cp /opt/phantomjs/phantomjs-1.9.8-linux-x86_64/bin/phantomjs /usr/bin/phantomjs
+wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-x86_64.tar.bz2 && \
```

Comment:
Can we put the indenting back. It's signifying a sub step in the RUN instruction